src/templateClickEventsHaveKeyEventsRule.ts

```diff
@@ -37,6 +43,21 @@ class TemplateClickEventsHaveKeyEventsVisitor extends BasicTemplateAstVisitor {
     if (!hasClick) {
       return;
     }
+
+    if (domElements.indexOf(el.name) === -1) {
```

Comment:
How many elements do we have in `domElements`? This linear check might be too slow. If we keep it as a set, the check would be `O(1)`.

Reply:
129 elements